### ISSUE TITLE
Move board from one instance to another.

### ISSUE THREAD

**Issue by daryl-o** [RestyaPlatform/board#4326]
What is the preferred method to move a board from one instance of restyaboard to another? I know there is an export board function but the import methods don't include "import from restyaboard".

**Comment by sridhar391** [comment]
@daryl-o Please follow the instructions from the documentation link https://restya.com/board/docs/backup-and-restore/#how-to-backup-and-restore-particular-board-or-all-boards-in-restyaboard to backup the board from one Restyaboard instance and restore it on the other Restyaboard instance

**Comment by daryl-o** [comment]
Thanks for the link to the scripts. I ran into issues with restyaboard_put.sh. json_decode was choking on the POST data (in server/php/bootstrap.php). Not sure if was the way I copy/pasted the script or some other problem. When I took out the newlines in the post data, everything seemed to work properly. For  example (combing the lines instead of how they were formatted nicely in the script):
``
  -d '{ "board_visibility": 0, "name": "'"${board_name}"'", "user_id": '${user_id}', "background_color": "'${board_background_color}'" }'
``

Documenting it here in case anybody else has the same issue as me.
